### PR TITLE
Update swagger documentation

### LINKED ISSUE #2635: Swagger errors
- Gitea version (or commit ref): 1ad902d5298202d5be14fd5a9c8ed6ce781a23c8
- Git version:
- Operating system:
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

There are errors in our swagger config, see https://online.swagger.io/validator/debug?url=https://try.gitea.io/swagger.v1.json

## Screenshots
image

### PR DESCRIPTION
**Pull request by ethantkoenig**
Fixes #2635. This PR fixes the following:
- Add docs for missing endpoints
- Add documentation for request parameters
- Make parameter naming consistent
- Fix response documentation

This PR has no functional change.

### CODE REVIEW

**Comment by lafriks** [comment]
LGTM

File: routers/api/v1/api.go [hunk 1]
```diff
@@ -1,15 +1,7 @@
-// Copyright 2015 The Gogs Authors. All rights reserved.
```
Comment (sapk) [inline routers/api/v1/api.go]: Why you remove copyrigths ? We could add gitea ones ?

File: routers/api/v1/api.go [hunk 2]
```diff
@@ -1,15 +1,7 @@
...
-// Use of this source code is governed by a MIT-style
 // license that can be found in the LICENSE file.
 
 // Package v1 Gitea API.
 //
-// This provide API interface to communicate with this Gitea instance.
```
Comment (sapk) [inline routers/api/v1/api.go]: Why you remove the description ?

**Comment by sapk** [comment]
Two little comments that should be fixed. For the rest LGTM. Thanks for doing it, it takes time and few good catch like https://github.com/go-gitea/gitea/pull/2899/files#diff-2620fdbeeb83ce43692534e6c2c39452R315.

**Comment by ethantkoenig** [comment]
@sapk Fixed. The "Terms of Service" link doesn't work, so that is still removed, everything else is restored.

File: routers/api/v1/api.go (hunk 1 above)
Comment (ethantkoenig) [reply routers/api/v1/api.go]: Good catch, fixed

File: routers/api/v1/api.go (hunk 2 above)
Comment (ethantkoenig) [reply routers/api/v1/api.go]: Good catch, fixed

**Pull request merged**